### PR TITLE
Add a deploy step to validate the metadata

### LINKED ISSUE #14: Add metadata validation step to deployment action
At a minimum, it should verify that the slug has been set.

### PR DESCRIPTION
**Pull request by palewire**
Fixes #14

### CODE REVIEW

File: .github/workflows/deploy-prod.yml [hunk 1]
```diff
@@ -10,9 +10,39 @@ on:
...
+          cache: 'npm'
+
+      - id: install-project-modules
+        name: Install project dependencies
+        run: npm ci --production
+        shell: bash
+
+      - id: validate-metadata
+        name: Validate metadata
+        if: github.repository != 'thecityny/fullpage-react-template'  # Skip this step when we're on the base template
```
Comment (sraby) [inline .github/workflows/deploy-prod.yml:38]: Nice!

**Review by sraby — APPROVED** [review]
Woo! this is great and an awesome example too for adding custom linter steps to our build processes. thanks a lot @palewire

**Comment by palewire** [comment]
Great. Should we try creating a new repo from the template and see if we can throw the error?

**Comment by sraby** [comment]
Yes! @palewire wanna give it a shot, or should I?

**Comment by palewire** [comment]
I think you should do it since it's probably better that the test run be within your organization.

**Comment by sraby** [comment]
@palewire looking goood!! https://github.com/thecityny/test-app/actions/runs/3587452227/jobs/6037815700

**Comment by sraby** [comment]
And @palewire when we change the `.env` it passes: https://github.com/thecityny/test-app/actions/runs/3587476510

**Comment by palewire** [comment]
You love to see it

**Pull request merged**